.gitignore

```diff
@@ -5,5 +5,6 @@
 #
 # https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
 
+/ci
```

Comment:
Do we really need this?

Reply:
This is to store the code coverage report for the CI. I can remove that from here and build the report in the root. Either is fine.

Reply:
Actually I just saw that you got the `build` directory there so this can be removed.